src/test/java/com/google/javascript/gents/singleTests/comments.js

```diff
@@ -44,3 +44,28 @@ var x;
 
 /** @export {number} */
 let m = 4;
+
+// head comment
+if (m) {
+  // comment at the bottom of if block
+}
+else {
+  // comment at the bottom of else block
+}
+
+var a = function() {
+  return {
+    b: function() {}
+  }
+};
+var c = function() {};
+
+// comment before GETPROP
+a(
+// comment in GETPROP
```

Comment:
The AST looks like this:
```
CALL 1 [length: 18]
    GETPROP 1 [length: 16]
        CALL 1 [length: 14] [free_call: 1]
            NAME a 1 [length: 1]
        STRING b 3 [length: 1]
```
So the next node (`STRING b 3 [length: 1]`) is in `GETPROP`.